### PR TITLE
Implement list users

### PR DESCRIPTION
**Pull request by igoihman**

### CODE REVIEW

**Comment by igoihman** [comment]
@vkareh @zgalor PTAL

File: cmd/ocm/list/user/cmd.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+		&args.step,
+		"step",
+		false,
+		"Load pages one step at a time",
+	)
+	fs.StringVar(
+		&args.columns,
+		"columns",
+		"id, name, api.url, openshift_version, cloud_provider.id, region.id, state",
+		"Specify which columns to display separated by commas, path is based on Cluster struct",
```
Comment (zgalor) [inline cmd/ocm/list/user/cmd.go:74]: This seems wrong. Should be user columns?

File: cmd/ocm/list/user/cmd.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	}
+
+	groups, err := c.GetGroups(clusterCollection, cluster.ID())
+	if err != nil {
+		return fmt.Errorf("Failed to get users for cluster '%s': %v", clusterKey, err)
+	}
+
+	// Create the writer that will be used to print the tabulated results:
+	writer := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
+	fmt.Fprintf(writer, "GROUP\t\tNAME\n")
```
Comment (vkareh) [inline cmd/ocm/list/user/cmd.go:112]: Should this be `NAME`, `USER`, or `USERNAME`? I have mixed feelings (but no strong opinions) :stuck_out_tongue: 

/nitpick

Comment (igoihman) [reply cmd/ocm/list/user/cmd.go]: changed to `USER`

File: cmd/ocm/list/user/cmd.go (hunk 1 above)
Comment (igoihman) [reply cmd/ocm/list/user/cmd.go]: oops :)

**Pull request merged**